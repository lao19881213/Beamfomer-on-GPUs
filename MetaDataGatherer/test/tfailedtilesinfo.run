#!/bin/sh

# Exit with skipped status if unable to reach RS005.
ping -c 1 RS005.astron.nl > /dev/null 2>&1 ||  exit 3

# Create the parset for this test.
# The start and end times have to match those in the SAS data base.
cat > tfailedtilesinfo_tmp.parset <<EOF
StartTime = 2011/05/15/16:59:08.0 
EndTime = 2012/10/21/22:59:06.0
BrokenTilesFile = tfailedtilesinfo_tmp.before
FailedTilesFile = tfailedtilesinfo_tmp.during
host = RS005.astron.nl
db = TESTLOFAR_4
port = 5432
EOF

# Get the failed tile info.
../src/failedtilesinfo tfailedtilesinfo_tmp.parset

# Print the output, so it will be compared by assay.
echo "Failed before:"
cat tfailedtilesinfo_tmp.before
echo
echo "Failed during:"
cat tfailedtilesinfo_tmp.during
