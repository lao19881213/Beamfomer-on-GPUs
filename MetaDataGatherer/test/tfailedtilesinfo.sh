#!/bin/sh
./runctest.sh tfailedtilesinfo
