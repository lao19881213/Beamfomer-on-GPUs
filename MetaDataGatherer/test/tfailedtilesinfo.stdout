Failed before:
CS013.RCU86. 2011-Mar-04 09:56:05.496000
CS026.RCU58. 2011-Mar-04 09:55:54.096000
CS026.RCU59. 2011-Mar-04 09:56:05.496000
CS026.RCU69. 2011-Mar-04 09:55:34.239000
RS208.RCU53. 2011-Mar-04 12:16:54.317000
RS208.RCU54. 2011-Mar-04 12:17:06.514000
RS208.RCU59. 2011-Mar-04 11:08:15.719000
RS208.RCU60. 2011-Mar-04 11:08:27.322000
RS208.RCU67. 2011-Mar-04 12:18:17.987000
RS208.RCU68. 2011-Mar-04 12:18:29.275000
RS503.RCU55. 2011-Mar-04 12:00:36.614000
RS503.RCU56. 2011-Mar-04 12:01:25.298000
RS503.RCU57. 2011-Mar-04 12:24:49.665000
RS503.RCU58. 2011-Mar-04 12:01:37.782000
RS503.RCU65. 2011-Mar-04 12:01:51.709000
RS503.RCU66. 2011-Mar-04 12:01:59.900000
RS503.RCU79. 2011-Mar-04 12:25:22.066000
RS503.RCU80. 2011-Mar-04 12:02:53.574000
RS503.RCU85. 2011-Mar-04 12:02:25.409000
RS503.RCU86. 2011-Mar-04 12:25:57.172000
RS503.RCU95. 2011-Mar-04 12:03:44.517000

Failed during:
CS003.RCU94. 2011-May-19 21:33:33.010000
RS106.RCU18. 2011-May-19 22:59:05.500000
CS101.RCU66. 2011-May-20 10:00:05
RS106.RCU0. 2011-May-20 10:00:05
CS001.RCU0. 2011-May-20 17:24:56.115000
